pkg/webhooks/pod_admission_webhook.go

```diff
@@ -82,7 +83,7 @@ func (p *podWebhook) leaderPodScheduled(ctx context.Context, pod *corev1.Pod) (b
 	}
 	scheduled := leaderPod.Spec.NodeName != ""
 	if !scheduled {
-		log.V(3).Info("leader pod %s is not yet scheduled", leaderPod.Name)
+		log.V(0).Info("leader pod %s is not yet scheduled", leaderPod.Name)
```

Comment:
```suggestion
		log.V(2).Info("leader pod %s is not yet scheduled", leaderPod.Name)
```

Either we should use level 2 or just remove the V(0) and use log.Info(...)

What log level do we deploy jobset with? I think 2 is usually default logging anyway.

Reply:
yeah, I think we generally have been using 2 I think

Reply:
Done